Feature/f 4

AniraUoren opened the pull request:
Сделан блок таски F-4 "Что входит в программу"

777toha commented on src/scss/program/program.scss:
```diff
@@ -1,15 +1,173 @@
+		padding: 0;
+		margin: 0;
+		margin-top: -8px;
+		max-width: 789px;
```
повторение ширины и макс-ширины, так не нужно

AniraUoren replied:
@777toha не очень поняла о каком повторении речь.

777toha replied:
у тебя указана макс ширина и ширина, это посути одно и тоже, я думаю такого стоит избегать для правильной работы

777toha commented on src/scss/program/program.scss:
```diff
@@ -1,15 +1,173 @@
+			&:nth-child(7) {
+				width: 305px;
+			}
+		}
```
по поводу чилдренов, не стоит каждому давать опредленную ширину, если текст измениться, верстка поедет

AniraUoren replied:
Делала для полного совпадения с макетом. Если настолько PP не нужно - уберу, но блок будет не совсем такой красивый.

777toha replied:
можно сделать, поэкспериментируй, просто посмотри
![image](https://github.com/EasyDo-project/easydo-webpack/assets/110735886/7b9f752f-2747-4470-8d63-931f24f3f85b)
просто задай общий размер и подели на два столба

777toha commented:
![image](https://github.com/EasyDo-project/easydo-webpack/assets/110735886/b550e9a3-97d2-41eb-a200-35af9b7cb831)

Не могу запустить твою ветку, отсутствуют файлы

777toha commented on src/pages/index.html:
```diff
@@ -13,9 +13,74 @@
+						<p class="program__paragraph">
+							<span class="program__highlight">Постоянная поддержка</span>
+							куратора онлайн
+						</p>
```
весь текст в константы и импортировать сюда

AniraUoren replied:
@777toha можно подробнее что ожидаем получить?
Нужно хранить текст в js и добавлять оттуда?

777toha replied:
да, создать папку /utils и там файл с константами, и от туда импортировать

AniraUoren committed:
fix: Правки в блок партнеров.

1. Добавлены файлы шрифтов.
2. Убран дублирующий стиль для изображения.
3. Убраны стили для каждого отдельного элемента списка.

Closes F-4

AniraUoren committed:
fix: Правки в блок партнеров.

4. Блок генерируется из переменной с помощью JS.

Closes F-4

AniraUoren commented:
Внесла правки. Посмотри, пожалуйста.

AniraUoren committed:
fix: Правки в блок партнеров.

4. Немного больше ООП.

Closes F-4

777toha commented:
![image](https://github.com/EasyDo-project/easydo-webpack/assets/110735886/b38d189e-d79e-4be3-baf3-4e4d131ee313)
Вот тут, сделай что бы картинку была по центру
![image](https://github.com/EasyDo-project/easydo-webpack/assets/110735886/c274d2a9-7698-43e2-9cf6-baa9011f758e)
вот тут небольшая проблема, наезжает картинка на текст

777toha commented:
![image](https://github.com/EasyDo-project/easydo-webpack/assets/110735886/f5acb292-1b9c-42ce-b621-4c2ebeec3198)
При пуле нужно смотреть что бы в консоли не было ошибок
это ошибки старых картинок которых больше нету и он не может их найти
ищи их где то тут
![image](https://github.com/EasyDo-project/easydo-webpack/assets/110735886/3c2ab9d7-23f3-4ba4-a675-196c0bcd01a7)

777toha commented on src/components/Importer.js:
```diff
@@ -0,0 +1,19 @@
+			.cloneNode(true);
+	}
+
+	generateProgramFeature() {
+		return this.getTemplate();
+	}
```
не совсем понял зачем тебе эта функция

777toha commented on src/pages/index.html:
```diff
@@ -13,9 +13,27 @@
+		<footer></footer>
+		<template id="programElement">
+			<li class="program__list-el">
+				<span class="program__counter">1</span>
```
нужна ли тут единица?

777toha commented:
в целом молодец, что сделала перебор по массиву и вынесла в константы

AniraUoren committed:
fix: Правки в блок.

1. Вернула на точке перелома 100% ширины картинке. Немного поправила чтобы смотрелось на других ширинах.
2. Почистила index.html от фавиконов и манифестов, которых нет.
3. Убрала лишнюю функцию которая была в Importer.js
4. Забыла подтереть единицу в index.html - убрала.

Closes F-5

AniraUoren commented:
Вернула 100% ширины картинке - помогает ей сохранять резиновость.
Остальное тоже поправила.

777toha committed:
fix merge
